assets/style.css

```diff
@@ -388,3 +388,9 @@ blockquote {
     border-color: var(--border-color);
     color: var(--foreground-secondary-color);
 }
+
+footer {
+    font-size: 0.9em;
+    color: #999;
```

Comment:
Each new change to the template should support both light and dark modes. That is why all of our color values are CSS variables. After I merged your last PR for blockquote, I changed all those CSS values to CSS variables and made sure they looked good for dark mode too.

Reply:
Ok, I will fix it later.